assembl/static2/js/app/components/administration/resourcesCenter/index.jsx

```diff
@@ -0,0 +1,136 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import arrayMutators from 'final-form-arrays';
+import { type ApolloClient, compose, withApollo } from 'react-apollo';
+import { Field } from 'react-final-form';
+
+import FieldArrayWithActions from '../../form/fieldArrayWithActions';
+import FileUploaderFieldAdapter from '../../form/fileUploaderFieldAdapter';
+import MultilingualTextFieldAdapter from '../../form/multilingualTextFieldAdapter';
+import MultilingualRichTextFieldAdapter from '../../form/multilingualRichTextFieldAdapter';
+import TextFieldAdapter from '../../form/textFieldAdapter';
+import { createResourceTooltip, deleteResourceTooltip } from '../../common/tooltips';
+
+import LoadSaveReinitializeForm from '../../../components/form/LoadSaveReinitializeForm';
+import { load, postLoadFormat } from './load';
+import { createMutationsPromises, save } from './save';
+import validate from './validate';
+import Loader from '../../common/loader';
+import SaveButton from '../../../components/administration/saveButton';
+
+type Props = {
+  client: ApolloClient,
+  editLocale: string,
+  lang: string
+};
+
+const loading = <Loader />;
+
+class ResourcesCenterAdminForm extends React.Component<Props> {
+  // componentDidMount() {
+  //   this.props.router.setRouteLeaveHook(this.props.route, this.routerWillLeave);
+  // }
+
+  // componentWillUnmount() {
+  //   this.props.router.setRouteLeaveHook(this.props.route, null);
+  // }
+
+  // routerWillLeave = () => {
+  //   if (this.dataHaveChanged() && !this.state.refetching) {
+  //     return I18n.t('administration.confirmUnsavedChanges');
+  //   }
+
+  //   return null;
+  // };
```

Comment:
Remove this commented code once you redo it for react-final-form.

Reply:
Thanks for the reminder ;-)